src/Uno.Sdk/packages.json

```diff
@@ -274,7 +274,10 @@
 			"Xamarin.AndroidX.Navigation.Fragment",
 			"Xamarin.AndroidX.Navigation.Runtime",
 			"Xamarin.AndroidX.Navigation.Common"
-		]
+		],
+		"versionOverride": {
+			"net10.0": "2.9.2.1"
+		}
```

Comment:
@kazo0 On the templates PR

<img width="315" height="59" alt="Image" src="https://github.com/user-attachments/assets/aac02842-32a7-406f-ad48-91d4af8bc191" />